pkg/nimble/netif/nimble_netif.c

```diff
@@ -644,6 +657,18 @@ int nimble_netif_accept(const uint8_t *ad, size_t ad_len,
     return NIMBLE_NETIF_OK;
 }
 
+int nimble_netif_accept(const uint8_t *ad, size_t ad_len,
+                        const struct ble_gap_adv_params *adv_params)
+{
+    return _accept(ad, ad_len, NULL, BLE_HS_FOREVER, adv_params);
+}
```

Comment:
A bit unrelated but why does this not have a timeout?

Reply:
Because when I designed this, there were stability issues with the timeout parameter in NimBLE. This is AFAIK fixed, but I did not dare to touch this API so far...